rhizome/host/lib/vm_setup.rb

```diff
@@ -83,8 +84,27 @@ def setup_networking(skip_persisted, gua, ip4, local_ip4, nics, ndp_needed, mult
     forwarding
   end
 
+  def unblock_ip4(ip4)
+    ip_net = NetAddr::IPv4Net.parse(ip4).network.to_s
+    filename = "/etc/nftables.d/#{q_vm}.conf"
+    temp_filename = "#{filename}.tmp"
+    File.open(temp_filename, File::RDWR | File::CREAT) do |f|
+      f.flock(File::LOCK_EX)
```

Comment:
should this be lock_nb as well? I presume we "never" anticipate interlocking on this file, but want to know if we have an aberrant situation.

Reply:
Yeah, I also think this would never require interlocking but I have added `lock_nb` to be on the safe side. Maybe it'll reveal a bug.